migrations/2023-06-20-191145_add_listingtype_sorttype_3_6_9_months_enums/down.sql

```diff
@@ -4,7 +4,7 @@ update local_user set default_sort_type = 'TopDay' where default_sort_type in ('
 -- rename the old enum
 alter type sort_type_enum rename to sort_type_enum__;
 -- create the new enum
-CREATE TYPE sort_type_enum AS ENUM ('Active', 'Hot', 'New', 'Old', 'TopDay', 'TopWeek', 'TopMonth', 'TopYear', 'TopAll', 'MostComments', 'NewComments');
+CREATE TYPE sort_type_enum AS ENUM ('Active', 'Hot', 'New', 'Old', 'TopHour', 'TopSixHour', 'TopTwelveHour', 'TopDay', 'TopWeek', 'TopMonth', 'TopYear', 'TopAll', 'MostComments', 'NewComments');
```

Comment:
I don't think this is right. These must be in the correct order, you've just added them in the middle randomly.

![Screenshot_2023-06-27-06-34-06-248_com termux](https://github.com/LemmyNet/lemmy/assets/930215/ec092160-13d8-4a12-955f-9e605e65f702)

Reply:
I'm sorry, I inserted them in the granularity order (i.e. hour < sixhours < twelvehours < day < etc.), will correct now